_includes/sidebar.html

```diff
@@ -45,6 +45,12 @@
                         <span>Resources</span>
                     </a>
                 </li>
+                <li class="sidebar-dropdown">
+                    <a href="branding.html">
+                        <i class="fas fa-palette"></i>
+                        <span>Branding</span>
+                    </a>
+                </li>
```

Comment:
Do we actually want a sidebar link for the branding page? It will only be of interest to a very small number of users. Should it be accessible via somewhere else?

Reply:
I can't think of anywhere else to put it, apart from maybe the About page?